node/toolrunner.ts

```diff
@@ -788,26 +788,22 @@ export class ToolRunner extends events.EventEmitter {
             this._debug('   ' + arg);
         });
 
-        options = this._cloneExecOptions(options);
-        if (!options.silent) {
-            options.outStream.write(this._getCommandString(options) + os.EOL);
+        const optionsNonNull = this._cloneExecOptions(options);
+        if (!optionsNonNull.silent) {
+            optionsNonNull.outStream.write(this._getCommandString(optionsNonNull) + os.EOL);
         }
 
-        let state = new ExecState(options, this.toolPath);
-        state.on('debug', (message) => {
+        let state = new ExecState(optionsNonNull, this.toolPath);
+        state.on('debug', (message: string) => {
             this._debug(message);
         });
 
-        let cp = child.spawn(this._getSpawnFileName(), this._getSpawnArgs(options), this._getSpawnOptions(options));
+        let cp = child.spawn(this._getSpawnFileName(), this._getSpawnArgs(optionsNonNull), this._getSpawnOptions(options));
 
         var stdbuffer: string = '';
         cp.stdout.on('data', (data: Buffer) => {
             this.emit('stdout', data);
 
-            if (!options.silent) {
```

Comment:
Why did you get rid of this?

Reply:
Bad merge -- I thought this had moved when I was resolving conflicts.